.github/workflows/conformance.yml

```diff
@@ -0,0 +1,85 @@
+run-name: Run ${{ inputs.test-suite }} test on Linux with Antrea ${{ inputs.antrea-version }} and K8s ${{ inputs.k8s-version }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      antrea-version:
+        description: The Antrea version to test. It could be a SHA-1 value, a branch, or a tag (e.g. a7b012b, release-1.12, v1.12.0). The main branch will be used if empty.
+        required: false
+      k8s-version:
+        description: The K8s version (e.g. v1.27.1) to test. Kind's default K8s version will be used if empty.
+        required: false
+      test-suite:
+        description: The test suite to run. Check run-k8s-e2e-tests.sh for which test cases these values represent.
+        type: choice
+        options:
+          - whole-conformance
+          - conformance
+          - network-policy
+          - sig-network
+          - all
+        default: whole-conformance
+        required: true
```

Comment:
[The example](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_dispatchinputs) set default and required at the same time.

Reply:
Got it, maybe it prevents users from unsetting it manually